src/app/_services/api/event.service.ts

```diff
@@ -14,4 +17,12 @@ export class EventService extends BaseService<EventDto> {
   constructor(httpClient: HttpClient, helperService: HelperService) {
     super(httpClient, helperService, '', constants.API_ROUTE_EVENTS, new Serializer());
   }
+
+  searchSpecific(filters: Filter[] = []): Observable<any> {
```

Comment:
instead of filters: Filter, please pass a uuid: string param. So you do not use it like this : filters[0].value

Reply:
Done.